pyproject.toml

```diff
@@ -79,22 +79,6 @@ test = [
     "pytest-cov",
     "pytest-subtests"
 ]
-dev = [
-    "ipython",
-    "myst-nb",
-    "readthedocs-sphinx-ext>=2.2",
-    "sphinx",
-    "sphinx-book-theme",
-    "sphinx-markdown-tables",
-    "sphinx-design",
-    "sphinx-mdinclude",
-    "mccabe>=0.6",
-    "pylint>=3.0",
-    "pytest",
-    "pytest-cov",
-    "pytest-subtests",
-    "pre-commit"
```

Comment:
`"pre-commit"` is neither part of `test` nor of `doc`

Reply:
I've renamed `test` to `dev` and added `pre-commit`